Testing/SystemTests/tests/framework/ISIS_PowderPolarisTest.py

```diff
@@ -580,15 +628,14 @@ def run_focus_no_absorption(per_detector=False):
     original_splined_path = os.path.join(input_dir, splined_file_name)
     shutil.copy(original_splined_path, spline_path)
 
-    inst_object = setup_inst_object(mode="PDF")
+    inst_object = setup_inst_object(mode=mode)
     return inst_object.focus(
         run_number=run_number,
         input_mode="Individual",
         do_van_normalisation=True,
         do_absorb_corrections=False,
         sample_empty=sample_empty,
         sample_empty_scale=sample_empty_scale,
-        van_normalisation_method="Relative",
```

Comment:
It's worth clarifying that the old reference file for `FocusTestNoAbsorption` is now used in the test `FocusTestNoAbsorptionWithRelativeNormalisation` - despite the fact that before this change the default `van_normalisation_method="Absolute"` when `mode="PDF"` (now it is "Relative")  - this is because the default vanadium normalisation was previously overwritten as a kwarg passed to `focus` here

Reply:
Sorry, no code change required! I think this comment on github is enough - just might be something that could confuse the eagle-eyed gatekeepers! In future probably worth adding this sort of thing to the PR description.